src/main/java/no/uib/inf112/core/RoboRally.java

```diff
@@ -48,22 +49,15 @@ public void create() {
         uiHandler = new UIHandler();
         new InputHandler();
         cpEventHandler = new ControlPanelEventHandler();
-
         map = new TiledMapHandler(FALLBACK_MAP_FILE_PATH);
-        player = new Player(5, 2, Direction.NORTH, false);
+        playerHandler = new PlayerHandler(numberOfPlayers);
+        playerHandler.generatePlayers(false); //Need this because of line 57 in player class
```

Comment:
Line 57 in Player will change over time. Does `headless` need explaining? Maybe we could add something to the wiki about how we test code using `headless` as parameters

Reply:
We could do that, but the reason i have a comment there is because normally generateplayers should be in the constructor, but that is not possible because of line 57 in player class, but we could probably do some refactoring to fix this.

Reply:
Then why not have the headless flag in the constructor?

Reply:
also having the generatePlayers exposed must eventually happen ([see this reply](https://github.com/inf112-v19/Blodige-roboter/pull/54#discussion_r257460078))

Reply:
It has nothing to do if it is headless or not, the problem occurs when you make a new playerhandler. If the genratePlayers() is in the constructor, the constructor of the player would execute this line `cards = RoboRally.getPlayerHandler().getDeck().draw(MAX_DRAW_CARDS);`, but as the playerhandler is not constructed yet, you will get a nullpointerexeption. So i don't see the point of having the headless flag in the constructor as it is never used there, and wont fix the problem.

Reply:
oh okay, but maybe say that in the comment and not the line number?

Reply:
I just don't like referencing line numbers as they will very quickly get out of date (you already forgot to update it to be line 56)

Reply:
And just so you know: you can run code next update with `Gdx.app.postRunnable(Runnable)`

Reply:
Agreed, but the comment was only there for the "reviwer" to understand that i had to do it a bit different. Line comments are only temporary from my part.